### PR TITLE
docker-images: Add ignite-ubuntu

### PR DESCRIPTION
**Pull request by efritz**
Verified to work when the image is created locally on the compute node containing firecracker.

```
docker build -t ignite-ubuntu-local .
sudo ignite create ignite-ubuntu-local --name test --cpus 1 --memory 1GB --size 4GB --ssh --runtime docker
sudo ignite test
sudo ignite exec test "docker run sourcegraph/lsif-go lsif-go -h"
```

### CODE REVIEW

File: docker-images/ignite-ubuntu/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ex
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+docker build -t "${IMAGE:-sourcegraph/ignite-ubuntu}" .
```
Comment (slimsag) [inline docker-images/ignite-ubuntu/build.sh:6]: Usually we would declare our own Dockerfile and have the same non-root username (`sourcegraph`) and UID/GID (`100:101`) that we use elsewhere - but from what I can tell ignite only runs as root and stores no data so probably not worth it

Comment (efritz) [reply docker-images/ignite-ubuntu/build.sh:6]: Definitely something I'll be revisiting once I ensure this entire effort will work.

File: docker-images/ignite-ubuntu/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM weaveworks/ignite-ubuntu:20.04-amd64@sha256:4f5f5ed56fae650ae122daa28a785192dda081be4f0b37dca2eb25ea57840500
```
Comment (slimsag) [inline docker-images/ignite-ubuntu/Dockerfile:1]: FYI: Ubuntu 20.04 does some weird stuff (switches to Snap package management), you may have trouble with dependencies in this version - maybe consider using 18.04

Comment (efritz) [reply docker-images/ignite-ubuntu/Dockerfile:1]: Will be able to easily swap this once I get unblocked with some of the more urgent deploy pipeline.

**Pull request merged**